### DISCUSSION TITLE
Ability to add tags to relevant records

### DISCUSSION THREAD

**Discussion by rohitgarud** [asreview/asreview#1291] [Ideas]
Is it possible to implement a functionality to add tags (or topic labels) to relevant papers, probably in the history section of the UI? The records can be filtered using these tags. This can help in clustering the records for the further stages of the systematic review. These tags can also be included in the exports. Is it a good idea or it will add any bias in further processing? To be clear, I am not suggesting using these tags to influence the AI models in any way, but rather just a helpful tool for researchers in the next stages of the systematic review. Thank You.

**Comment by rohitgarud** [comment]
I am planning to add some functionality to the [asreview-notes-export extension](https://github.com/rohitgarud/asreview-notes-export), where tags can be added to the notes and maybe parsed while exporting to csv for better filtering.  Suggestions are welcome.

**Comment by Rensvandeschoot** [comment]
A PR has been opened for adding the option to add tags to the review screen: https://github.com/asreview/asreview/pull/1527.
